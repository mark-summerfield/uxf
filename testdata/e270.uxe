uxf 1
[<a]
