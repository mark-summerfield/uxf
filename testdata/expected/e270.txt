uxf:E270:e270.uxe:2:unterminated string
uxf:E446:e270.uxe:2:expected list vtype, got a
